src/lambdaisland/ornament.cljc

```diff
@@ -226,7 +226,13 @@
          rule
 
          (simple-keyword? rule)
-         (second (class-name->garden (name rule)))
+         (let [girouette-garden (class-name->garden (name rule))]
+           (if (and (record? girouette-garden)
+                    (= (:identifier girouette-garden) :media))
+             (-> girouette-garden
+                 (update-in [:value :rules] (fn [rules]
+                                              (map #(into [:&] (rest %)) rules))))
```

Comment:
Could probably have been `assoc % 0 :&` but into looks cleaner to me for some reason :see_no_evil: